atlas/templates/cmd/server/config.go.gotmpl

```diff
@@ -29,6 +29,12 @@ const (
 	defaultDatabaseSSL      = "disable"
 	defaultDatabaseOption   = ""
 
+	defaultDatabaseOpenConn    = 10
+	defaultDatabaseMaxIdleConn = 10
+	defaultDatabaseMaxConnLifetime = 30 * time.Minute
+	defaultDatabaseMaxConnIdleTime = 10 * time.Minute
```

Comment:
@kd7lxl Do these defaults look good?

Reply:
Could be even shorter without adding significant overhead, but these are good.